### PR TITLE
re-enable WebRTC, unshorten all links, fixes #86 #244

### LINKED ISSUE #86: Consider enabling WebRTC in the future
See #84.

### PR DESCRIPTION
**Pull request by nodiscc**

### CODE REVIEW

**Comment by pyllyukko** [comment]
See <https://github.com/pyllyukko/user.js/issues/86#issuecomment-287873061>

**Comment by nodiscc** [comment]
Updated.

File: README.md [hunk 1]
```diff
@@ -131,11 +131,11 @@ HTML5 / [APIs](https://wiki.mozilla.org/WebAPI) / [DOM](https://en.wikipedia.org
...
 * Disable web notifications [ [1](https://support.mozilla.org/t5/Firefox/I-can-t-find-Firefox-menu-I-m-trying-to-opt-out-of-Web-Push-and/m-p/1317495#M1006501) ]
 * Make sure the User Timing API does not provide a new high resolution timestamp [ [1](https://trac.torproject.org/projects/tor/ticket/16336) ]
-* Disable Web Audio API [ [1](https://bugzil.la/1288359) ]
+* Disable Web Audio API [ [1](https://bugzilla.mozilla.org/show_bug.cgi?id=1288359) ]
 * Disable Location-Aware Browsing (geolocation) [ [1](https://www.mozilla.org/en-US/firefox/geolocation/) ]
 * Disable raw TCP socket support (mozTCPSocket) [ [1](https://trac.torproject.org/projects/tor/ticket/18863) [2](https://www.mozilla.org/en-US/security/advisories/mfsa2015-97/) [3](https://developer.mozilla.org/docs/Mozilla/B2G_OS/API/TCPSocket) ]
 * Whether JS can get information about the network/browser connection
-* Disable WebRTC entirely
+* Disable WebRTC entirely to prevent leaking internal IP addresses (Firefox <52)
```
Comment (fmarier) [inline README.md]: If I understand these prefs correctly and you have `media.peerconnection.ice.default_address_only` in Firefox < 52, your internal IP address shouldn't be available to WebRTC.

Comment (nodiscc) [inline README.md]: I agree this is confusing. https://wiki.mozilla.org/Media/WebRTC/Privacy: There are 3 ways to plug the WebRTC LAN IP address leak, depending on Firefox version:

 * Firefox 18-42: Disable WebRTC entirely
 * Firefox 42-50: `media.peerconnection.ice.default_address_only: true`
 * Firefox >=51: `media.peerconnection.ice.no_host: true`

**Pull request merged**